### DISCUSSION TITLE
How to compare the size of a `Date` data

### DISCUSSION THREAD

**Discussion by SnowingFox** [react-hook-form/react-hook-form#8098] [Q&A] [answered]
Like this

```jsx
<input {...register('startTime', {
    min: Date.now() + 1000 * 60 * 30 // after half an hour
})} />
```
if i want to relize this, what should i do?

**Comment by bluebill1049** [comment, marked as answer]
> if i want to relize this, what should i do?

for custom behavior you can always fall back to `validate` function

**Reply by Moshyfawn** [reply]
Yep, there's not native HTML attribute to validate the date type input

**Reply by SnowingFox** [reply]
maybe that's the only way...

**Reply by SnowingFox** [reply]
```jsx
validate: () => getTime(new Date(getValues().endTime)) - getTime(new Date(getValues().startTime)) >= 0,
```
i did this, and it works well
